portfolio/src/main/webapp/index.html

```diff
@@ -27,6 +27,22 @@ <h2>About me</h2>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <div id="tic">
+        <h2>Tic-Tac-Toe</h2>
+        <button id="ticButton">Play</button>
+<!--         <div class="paper">
+          <div class="box"><span style="font-size : 300%">O</span></div>
+          <div class="box">2</div>
+          <div class="box"></div>
+          <div class="box"></div>
+          <div class="box"></div>
+          <div class="box"></div>
+          <div class="box"></div>
+          <div class="box"></div>
+          <div class="box"></div>
+      </div> -->
```

Comment:
This can be removed as the commented section is taking out space but not contributing anything.